### PR TITLE
Joflynn collectd plugins update

### PR DESCRIPTION
**Pull request by JoanneOFlynn2018**
Updating collectd plugins for 13. Some plugins in the update require configurations.

### CODE REVIEW

File: common/collectd/ref_collectd-plugins.adoc [hunk 1]
```diff
@@ -27,18 +27,31 @@
 [[appe-saf-collectd-plugins]]
 # collectd plugins
 
-This section contains a complete list of collectd plugins and configurations.
+This section contains a complete list of collectd plug-ins and configurations.
```
Comment (csibbitt) [inline common/collectd/ref_collectd-plugins.adoc]: This is undoubtedly correct, but I have to disagree anyways.

https://collectd.org/wiki/index.php/Table_of_Plugins

https://collectd.org/wiki/index.php?search=plug-in&title=Special%3ASearch&go=Go

Comment (JoanneOFlynn2018) [reply common/collectd/ref_collectd-plugins.adoc]: Chris, I like the way you think! You are correct. It should read: This section contains a complete list of collectd plugins and configurations for OpenStack Queens. The plugins list *should* be correct here but do let me know if you think I'm missing anything. The configurations for some of the plugins are still missing.  

In my opinion, it should indeed be plugin not plug-in, but our IBM style guide says plug-in, unfortunately :(

Comment (leifmadsen) [reply common/collectd/ref_collectd-plugins.adoc]: Sounds good to me. The wording proposed sounds much better.

Comment (csibbitt) [reply common/collectd/ref_collectd-plugins.adoc]: To be honest I only thought about "plugin" vs "plug-in" (overruled by the style guide, hah!), but I definitely agree on expanding the wording to clarify that it's the ones in OpenStack, not the sum total of collectd plugins. :+1:

**Pull request merged**